CHANGELOG.md

```diff
@@ -1,18 +1,10 @@
-# Release 0.23.0-dev
+# Release 0.27.0
 
 ### New features since last release
 
 * Adds Python 3.10 support.
```

Comment:
So we never actually released a "python 3.10 support version"?

Reply:
guess not! better late than never